Create Executive Vote - August 2, 2023.md (#2374)
makerdao/community: governance/votes/Executive Vote - August 2, 2023.md

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

### Enhanced Dai Savings Rate (DSR) Activation

As per this successful governance [poll](https://vote.makerdao.com/polling/QmcTRPLx), the Stability Scope has been amended to include an [Enhanced Dai Savings Rate (EDSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-). To achieve this, the DSR will be updated if this executive proposal passes.

- Increase the DSR by 4.81% from 3.19% to **8%**.

Please review the forum [thread](https://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-implement-enhanced-dsr/21405) for more information about this change.

### Spark Dai Direct Deposit Module (D3M) Parameter Change

Per this successful governance [poll](https://vote.makerdao.com/polling/QmSLj3HS), the Spark Protocol D3M will be updated if this executive proposal passes.

> - Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 180 million DAI from 20 million DAI to **200 million DAI**.

Reviewer: ```suggestion
- The Spark Protocol D3M (DIRECT-SPARK-DAI) [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased by 180 million DAI from 20 million DAI to **200 million DAI**.
```

> The [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) and the [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) will remain unchanged at 20 million DAI and 8 hours respectively.

Reviewer: Not sure you really need this? I guess it doesn't hurt.

Author: I'm anticipating being asked to include it by technical, agree it's largely redundant

Please review the forum [thread](https://forum.makerdao.com/t/phoenix-labs-proposed-changes-for-spark/21422) for more information about this change.

### Harbor Trade Credit (RWA004-A) Changes

> As per this successful governance [poll](https://vote.makerdao.com/polling/QmR8cYb1), the following changes to the Harbor Trade Credit (RWA004-A) vault will be made if this executive proposal passes.

Reviewer: The poll doesn't list the `tell` call?

Author: No but it's a technical requirement to achieve what we want on the centrifuge end, it's another one of these things that I only found out about after the poll was live, I made a comment in the thread to that effect

Author: It has the support of SF, Dewiz, Centrifuge and HTC to call it - I see no reason to block it tbh

> - Decrease the RWA004-A [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) by 7 million DAI from 7 million DAI to **0 DAI**.

Reviewer: ```suggestion
- Decrease the RWA004-A [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) from 7 million DAI to **zero DAI**.
```

Author: disagree on this one tbh, will resolve without changing

> - `tell` will be called on the RWA Liquidation Oracle for the RWA004-A vault. This will place the vault into a soft liquidation state. This is a technical requirement for Centrifuge to commence the unwinding process.

Reviewer: Not sure we should even be including this given that we didn't explicitly poll on 'Should we liquidate RWA004-A?'. Setting the debt ceiling to zero isn't the same as triggering liquidation.

Author: see above - it's not hard liquidation, the vault balance will remain unchanged, it's a technical requirement for centrifuge to do what they need to do

Author: It has the support of SF, Dewiz, Centrifuge and HTC to call it - I see no reason to block it tbh

Please review the forum [thread](https://forum.makerdao.com/t/request-to-poll-decrease-debt-ceiling-for-harbor-trade-credit-htc-drop-to-0/21373) for more information about this change.

### New Silver (RWA002-A) Doc Update

> Per this governance [poll](https://vote.makerdao.com/polling/QmaU1eaD), the `doc` parameter for RWA002-A will be updated if this executive proposal passes.

Reviewer: Use human-readable parameter language. So RWA Agreement versus doc, and link to docs. I agree that the name of that one is bad though.

Reviewer: I think you need to give more information here in general. Like, it's not just a doc update, it's agreement to restructure stuff, other parameter changes will come soon, etc.

> Per this governance [poll](https://vote.makerdao.com/polling/QmaU1eaD), the `doc` parameter for RWA002-A will be updated if this executive proposal passes.

Reviewer: ```suggestion
Per this governance [poll](https://vote.makerdao.com/polling/QmaU1eaD), the `doc` parameter for the New Silver vault will be updated if this executive proposal passes.
```

> - Update the RWA002-A `doc` to `QmTrrwZpnSZ41rbrpx267R7vfDFktseQe2W5NJ5xB7kkn1` - the new `doc` may be viewed [here](https://gateway.pinata.cloud/ipfs/QmTrrwZpnSZ41rbrpx267R7vfDFktseQe2W5NJ5xB7kkn1).

Reviewer: ```suggestion
- The RWA002-A `doc` will be updated to `QmTrrwZpnSZ41rbrpx267R7vfDFktseQe2W5NJ5xB7kkn1` - the new `doc` may be viewed [here](https://gateway.pinata.cloud/ipfs/QmTrrwZpnSZ41rbrpx267R7vfDFktseQe2W5NJ5xB7kkn1).
```

Reviewer: Same comments as above on the human-readable thing.

> Once the `doc` has been updated, the other approved changes, including Debt Ceiling and Stability Fee changes, can be actioned.

Reviewer: Oh it's down here.

Reviewer: Still think you need more detail on the context around the changes.

Author: reworded this bit

> For more information on the RWA Agreement (`doc`) parameter, please see the documentation [here](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement).

Reviewer: Just link it in-line on the first use, imo.

| David Phelps     | [0xd56e3E325133EFEd6B1687C88571b8a91e517ab0](https://etherscan.io/address/0xd56e3E325133EFEd6B1687C88571b8a91e517ab0)  | Composable          | 8.94         |
| seedlatam.eth    | [0x0087a081a9b430fd8f688c6ac5dd24421bfb060d](https://etherscan.io/address/0x0087a081a9b430fd8f688c6ac5dd24421bfb060d)  | Sovereign Finance   | 11.92        |
| StableLab        | [0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08](https://etherscan.io/address/0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08)  | Growth              | 14.9         |
| flipsidegov      | [0x300901243d6CB2E74c10f8aB4cc89a39cC222a29](https://etherscan.io/address/0x300901243d6cb2e74c10f8ab4cc89a39cc222a29)  | Growth              | 14.9         |

This comes to a total of **134.1 MKR**.

### DAO Resolution

As per this forum [post](https://forum.makerdao.com/t/clydesdale-quarterly-return-of-surplus-fund/21291) that has been approved by the Stability Scope Facilitators, the following DAO Resolution will be approved if this executive proposal passes.

> - Approve the DAO Resolution with the IPFS hash `QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p` - the DAO Resolution may be viewed [here](https://gateway.pinata.cloud/ipfs/QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p).

Reviewer: ```suggestion
- The DAO Resolution with the IPFS hash `QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p` will be approved by MakerDAO - the DAO Resolution may be viewed [here](https://gateway.pinata.cloud/ipfs/QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p).
```

> In summary, this will instruct the trustees of the Monetalis Clydesdale (RWA007-A) JAT1 arrangement to sell a portion of ETF holdings and return profits to MakerDAO via the input conduit.

Reviewer: ```suggestion
In summary, this will instruct the trustees of the Monetalis Clydesdale (RWA007-A) JAT1 arrangement to sell a portion of ETF holdings and return the profits to the MakerDAO surplus buffer via the input conduit smart contract.
```

### Launch Project Funding

> As per this forum [post]($TBD) from the Accessibility Scope Facilitators, as per the [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget), the following DAI transfer to support the Launch Project will take place, if this executive proposal passes.

Reviewer: ```suggestion
As per this forum [post]($TBD) from the Accessibility Scope Facilitators and the [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget), the following DAI transfer from the Launch Project budget will take place, if this executive proposal passes.
```

Reviewer: Something like that?

> - Transfer **2,000,000 DAI** to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).

Reviewer: ```suggestion
- 2 million DAI will be transferred to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
```

Reviewer: I guess you'll add more information here once we have a forum post that says where it's going? lol

### Spark Proxy Spell

As part of this executive proposal, a Proxy Spell for Spark Protocol at [0x443f3f4328553f5f85dFc0BA3D59969708201E14](https://etherscan.io/address/0x443f3f4328553f5f85dFc0BA3D59969708201E14) will be triggered if this executive proposal passes.

> This will include the following changes to Spark Protocol:

Reviewer: ```suggestion
This Proxy Spell includes the following changes to Spark Protocol:
```

#### DAI InterestRateStrategy Update

> - Deploy a new DAI_interestRateStrategy to tie the borrow rate of Dai to the ETH-C Stability Fee. This is required due to the activation of the EDSR, as vault stability fees and other rates will continue to be based on the underlying DSR rather than the EDSR. The ETH-C stability fee equals the underlying DSR, making it an appropriate reference for Spark Protocol.

Reviewer: ```suggestion
Deploy a new DAI_interestRateStrategy to tie the borrow rate of Dai to the ETH-C Stability Fee. This is required due to the activation of the EDSR, as vault stability fees and other rates will continue to be based on the underlying DSR rather than the EDSR. The ETH-C stability fee equals the underlying DSR, making it an appropriate reference for Spark Protocol.
```

Is the new DAI strategy actually deployed as part of the Proxy Spell?
Maybe link to Stability Fee docs again.

Not sure the explanation as written is going to help anyone that doesn't already know why it's happening, to be honest. It's moderately difficult to explain, admittedly.

#### Spark Protocol Parameter Changes

> Per this successful governance [poll](https://vote.makerdao.com/polling/QmZyFH21), the following parameter changes will be made to Spark Protocol.

Reviewer: I think you might need a little more summary here. The parameter changes look broken otherwise.

Author: added explainers from Spark team

> - Decrease the DAI market [Loan-To-Value (LTV)](https://docs.aave.com/risk/asset-risk/risk-parameters#loan-to-value) from 74% to **0.01%**.

Reviewer: ```suggestion
- The DAI market [Loan-To-Value (LTV)](https://docs.aave.com/risk/asset-risk/risk-parameters#loan-to-value) will be decreased from 74% to **0.01%**.
```

> - Decrease the DAI market [Liquidation Threshold](https://docs.aave.com/risk/asset-risk/risk-parameters#liquidation-threshold) from 76% to **0.01%**.

Reviewer: ```suggestion
- The DAI market [Liquidation Threshold](https://docs.aave.com/risk/asset-risk/risk-parameters#liquidation-threshold) will be decreased from 76% to **0.01%**.
```

> - Decrease the WETH market [Reserve Factor](https://docs.aave.com/risk/asset-risk/risk-parameters#reserve-factor_) by 10% from 15% to **5%**.

Reviewer: ```suggestion
- The WETH market [Reserve Factor](https://docs.aave.com/risk/asset-risk/risk-parameters#reserve-factor_) will be decreased by 10% from 15% to **5%**.
```

> - Decrease the WETH market [varialeRateSlope1](https://docs.aave.com/risk/liquidity-risk/borrow-interest-rate#variable-interest-rate-model-parameters) by 0.8% from 3.8% to **3%**.

Reviewer: ```suggestion
- The WETH market [varialeRateSlope1](https://docs.aave.com/risk/liquidity-risk/borrow-interest-rate#variable-interest-rate-model-parameters) will be decreased by 0.8% from 3.8% to **3%**.
```
